pallet/staking/src/mock.rs

```diff
@@ -228,6 +229,8 @@ frame_support::parameter_types! {
 impl darwinia_staking::Config for Runtime {
 	type Deposit = Deposit;
 	type Kton = KtonStaking;
+	#[cfg(feature = "runtime-benchmarks")]
+	type KtonHelper = KtonMinting;
```

Comment:
Why don't use `type Kton = KtonMinting` in the pallet-deposit directly?

Reply:
Because they are different interfaces/traits.

Reply:
What's the difference between them? Seems both implement the `darwinia_deposit::SimpleAsset`.